Android_Development_with_Kotlin/07. View Groups/07.2 Relative Layout in Kotlin.md

```diff
@@ -0,0 +1,141 @@
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:paddingLeft="10dp"
+    android:paddingRight="10dp">
+     
+    <TextView
+        android:id="@+id/textView1"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentLeft="true"
+        android:text="First name:"
+        android:layout_marginTop="20dp"
+        android:textSize="20dp"/>
+  
+    <EditText
+        android:id="@+id/editText1"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_toRightOf="@id/textView1"
+        android:layout_marginTop="8dp"/>
+      
+    <TextView
+        android:id="@+id/textView2"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentLeft="true"
+        android:layout_below="@+id/textView1"
+        android:layout_marginTop="10dp"
+        android:text="Last name:"
+        android:textSize="20dp"/>
+  
+    <EditText
+        android:id="@+id/editText2"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_centerHorizontal="true"
+        android:layout_toRightOf="@id/textView2"
+        android:layout_marginTop="45dp"/>
+  
+    <Button
+        android:id="@+id/btn4"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_centerHorizontal="true"
+        android:layout_below="@id/textView2"
+        android:layout_marginTop="20dp"
+        android:text="Submit" />
+      
+</RelativeLayout>
+```
+
+## <div align="center">🧠MainActivity.kt file</div>
+When we have created layout, we need to load the XML layout resource from our activity onCreate() callback method and access the UI element form the XML using findViewById.
+
+
+<pre><code>
+package com.geeksforgeeks.myfirstKotlinapp
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+  
+  
+class MainActivity : AppCompatActivity() {
+  
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+  
+        // below access the UI elements
+  
+  
+    }
+}
+</code></pre>
+
+## <div align="center">🎭RelativeLayout Output:</div>
+We can run the application using the Android Virtual Device(AVD) to get the output of the above code.
+
+<p align ="center"><img src="https://media.geeksforgeeks.org/wp-content/uploads/20191101172521/Screen-Shot-2019-11-01-at-5.25.00-PM.png" alt="mobile" height=450px width=230px></p>
+
+So this was all that you needed to know about the Relative Layouts with Kotlin.
+## HAPPY LEARNING 🙌😎
+<hr>
+
+## REFERENCES-:
+
+- For preparing these documentation official documentation of <a href="https://kotlinlang.org/docs/home.html">kotlin</a> was reffered.

```

Comment:
Fixed grammatical error, and spelling mistakes.

Reply:
@rahulsain please review again. Fixed all the issues.